src/components/Marketplace/MarketplaceScreen.js

```diff
@@ -78,33 +82,50 @@ export default function MarketplaceScreen() {
             produceType,
             palletsAvailable: record.fields['pallets available'] || 0,
             hasAgencyPrice,
+            // fields for sorting
+            'sell by date': record.fields['sell by date'] || 0,
+            'first available date': record.fields['first available date'],
+            'available until': record.fields['available until'],
+            'date entered': record.fields['data entered'],
+            'produce name': record.fields['produce name'],
           };
           processedProduceRecords.push(recordInfo);
         });
         setProduceListings(processedProduceRecords);
       });
   }
 
+  // Passed to sidebar and sort menu component, tracks current sort option
+  const [sortOrder, setSortOrder] = useState('sell by date');
+
   // Manages farm season filtering (fx, filter options, # of items per option)
-  const [seasonFilters, setSeasonFilters] = useState([]);
+  const [checkedSeasonFilters, setCheckedSeasonFilters] = useState([]);
   // Called by child comp when season filters changed, sets new filters or empty [] if none/reset
   const onSeasonFilterChange = (newFilters) => {
-    setSeasonFilters(newFilters);
+    setCheckedSeasonFilters(newFilters);
   };
   const farmSeasonFilters = ['Fall', 'Winter', 'Summer', 'Spring'];
   const [itemsPerFarmSeason, setSeasonItems] = useState([]);
 
+  // Manages item type filtering
+  const [checkedItemTypes, setCheckedItemTypes] = useState([]);
+  const onItemFilterChange = (newFilters) => {
+    setCheckedItemTypes(newFilters);
+  };
+  const itemTypeFilters = ['Agency Price', 'Standard Items'];
+  const [itemsPerItemType, setItemsPerItemType] = useState([]);
```

Comment:
Maybe define all the state variables together at the top of this func? Just to make it consistent with how you've defined them in the function above.